cranelift/codegen/src/context.rs

```diff
@@ -366,7 +366,7 @@ impl Context {
     /// "store-to-load forwarding").
     pub fn replace_redundant_loads(&mut self) -> CodegenResult<()> {
         let mut analysis = AliasAnalysis::new(&mut self.func, &self.domtree);
```

Comment:
I think this doesn't need to be `&mut` any more:
```suggestion
        let mut analysis = AliasAnalysis::new(&self.func, &self.domtree);
```

It's not important since the next line still has to take a mutable borrow, but still.

Reply:
Fixed, thanks!